jekyll/_includes/snippets/macos-resource-table.md

```diff
@@ -4,10 +4,13 @@ medium | 4 @ 2.7 GHz | 8GB
 macos.x86.medium.gen2 | 4 @ 3.2 GHz | 8GB
 large | 8 @ 2.7 GHz | 16GB
 macos.x86.metal.gen1 | 12 @ 3.2 GHz | 32GB
+macos.m1.large.gen1 | 8 | 12GB
 {: class="table table-striped"}
 
+The `medium` and `large` resource classes are being deprecated on October 2, 2023. See our [announcement](https://discuss.circleci.com/t/macos-resource-deprecation-update/46891) for more details.
```

Comment:
It might be worth calling out that images will no longer be provided for these resource classes moving forward. Some customers may confuse this and think that images will still be provided up to October 2023

Reply:
Good point. I added it below the supported Xcode table, but over-communicating is probably best in this case.

Reply:
Updated.